src/nccl_ofi_ofiutils.c

```diff
@@ -245,7 +245,7 @@ int nccl_ofi_ofiutils_get_providers(const char *prov_include,
 }
 
 int nccl_ofi_ofiutils_init_connection(int api_version, struct fi_info *info, struct fid_domain *domain,
-				      struct fid_ep **ep, struct fid_av **av, struct fid_cq **cq)
+				      struct fid_ep **ep, struct fid_av **av, struct fid_cq **cq, bool tagged)
```

Comment:
Why do we need an argument to determine this? Can we not deduce this from the info object (which would have requested tagged capabilities in the sendrecv protocol but not anymore in the RDMA protocol with this change)?

Reply:
good point, I will do so.